index.html

```diff
@@ -40,201 +43,227 @@
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <header class="links-text">Let's connect</header>
+        <ul class="icons-section">
           <li>
             <a href="https://twitter.com/Lucas_David_22">
-              <img src="./svg_files/twitter.svg" alt="twitter" />
-            </a>
+              <img src="./svg_files/twitter.svg" alt="twitter"/></a>
           </li>
           <li>
             <a href="https://www.linkedin.com/in/lucas-erkana-b30a0b3b/">
-              <img src="./svg_files/Linkedin.svg" alt="Linkedin"/>
-            </a>
+              <img src="./svg_files/Linkedin.svg" alt="Linkedin"/></a>
           </li>
           <li>
             <a href="https://medium.com/">
-              <img src="./svg_files/medium.svg" alt="medium"/>
-            </a>
+              <img src="./svg_files/medium.svg" alt="medium"/></a>
           </li>
           <li>
             <a href="https://github.com/Lucash2022">
-              <img src="./svg_files/github.svg" alt="github"/>
-            </a>
+              <img src="./svg_files/github.svg" alt="github"
+            /></a>
           </li>
           <li>
             <a href="https://www.angellist.com/">
               <img src="./svg_files/angellist.svg" alt="angellist" />
             </a>
           </li>
-      </ul>
-      <div class="spacer"></div>
-      <button class="btn" type="button">Get my resume</button>
-      <div class="spacer"></div>
-      <ul class="about_section_lists">
-            <div class="dropdown"> 
-                <li class="section_header">Languages</li><span>^</span>
+        </ul>
+      </div>
+    </section>
+    <section class="project_section">
+      <!-- first project -->
+      <div class="first_project">
+        <div class="Epxlore_image">
+           <img class="Nature" src="./svg_files/Nature.png" alt="Nature"  />
+        </div>
+        <div class="tonic_section">
+          <h2>Tonic</h2>
+          <div class="flex_list">
+                  <ul class="list">
+                    <h5>CANOPY</h5>
+                    <li class="list_items">Back End Dev</li>
+                    <li class="list_items">2015</li>
+                  </ul>
+                  <p class="project_description">
+                     A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+                  </p>
+                  <div class="tags">
+                    <ul>
+                      <li><a href="" class="tag">html</a></li>
+                      <li><a href="" class="tag">css</a></li>
+                      <li><a href="" class="tag">javascript</a></li>
+                    </ul>
+                  </div>
+                  <button class="btn" type="button">See Project</button>
```

Comment:
[ Required ] Please ensure that the Mobile version of your portfolio is not affected by the `css` styling for the desktop version. It appears that on smaller screens of width `375px` your design is not consistent with the Figma template. Please see the screenshot below for more details.

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/200015160-74f08b9b-37c5-4a0f-9893-d47290b1675f.png" width="300px" />



## Your implementation of the design
<img src="https://user-images.githubusercontent.com/16392046/200014778-7f2e1f22-fb6d-4093-a037-8acf0df6e5cc.png" width="300px" />